delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/AccessUrl.scala

```diff
@@ -108,4 +110,8 @@ object AccessUrl {
   private def context(mappings: ApiMappings): JsonLdContext =
     JsonLdContext(ContextValue.empty, prefixMappings = mappings.prefixMappings, aliases = mappings.aliases)
 
+  private val defaultMappings: ApiMappings = ApiMappings(
+    Map("_" -> schemas.resources, "schema" -> schemas.shacl, "resolver" -> schemas.resolvers)
```

Comment:
I think these used to be defined as plural (`schemas`, `resolvers`)...

Reply:
No, it is as I defined: https://github.com/BlueBrain/nexus/blob/v1.4.x/delta/src/main/scala/ch/epfl/bluebrain/nexus/kg/directives/ProjectDirectives.scala#L24

Reply:
right, please ignore